### ISSUE TITLE
VirtualMachine is in crashingbackoff state 

### ISSUE THREAD

**Issue by hjxgitcode** [kubevirt/kubevirt#11226]
I am according https://kubevirt.io/labs/kubernetes/lab1.html Create testvm.But a few minutes,virt-launcher pod is Error.

![1](https://github.com/kubevirt/kubevirt/assets/158697049/43a15fbe-f7c0-46dc-8259-d3539adfcddc)

Environment:
- KubeVirt version (use virtctl version): v1.1.1
- Kubernetes version (use kubectl version): v1.28.2
-  k8s distro : kubeadm
- OS: CentOS-7-x86_64-DVD-1804.iso

**virt-launcher  log:**
```
{"component":"virt-launcher-monitor","level":"error","msg":"could not create up serial console term file: /var/run/kubevirt-private/fd047462-f1b5-429c-98ac-f31dd30e1356/virt-serial0-log-sigTerm","pos":"virt-launcher-monitor.go:72","reason":"open /var/run/kubevirt-private/fd047462-f1b5-429c-98ac-f31dd30e1356/virt-serial0-log-sigTerm: no such file or directory","timestamp":"2024-02-13T08:00:57.677833Z"}
{"component":"virt-launcher","level":"info","msg":"Collected all requested hook sidecar sockets","pos":"manager.go:86","timestamp":"2024-02-13T08:00:57.724043Z"}
{"component":"virt-launcher","level":"info","msg":"Sorted all collected sidecar sockets per hook point based on their priority and name: map[]","pos":"manager.go:89","timestamp":"2024-02-13T08:00:57.724175Z"}
{"component":"virt-launcher","level":"info","msg":"Connecting to libvirt daemon: qemu+unix:///session?socket=/var/run/libvirt/virtqemud-sock","pos":"libvirt.go:563","timestamp":"2024-02-13T08:00:57.724739Z"}
{"component":"virt-launcher","level":"info","msg":"Connecting to libvirt daemon failed: virError(Code=38, Domain=7, Message='Failed to connect socket to '/var/run/libvirt/virtqemud-sock': No such file or directory')","pos":"libvirt.go:571","timestamp":"2024-02-13T08:00:57.725578Z"}
{"component":"virt-launcher","level":"info","msg":"libvirt version: 9.5.0, package: 6.el9 (builder@centos.org, 2023-08-25-08:53:56, )","subcomponent":"libvirt","thread":"38","timestamp":"2024-02-13T08:00:57.744000Z"}
{"component":"virt-launcher","level":"info","msg":"hostname: testvm","subcomponent":"libvirt","thread":"38","timestamp":"2024-02-13T08:00:57.744000Z"}
{"component":"virt-launcher","level":"error","msg":"Unable to open /dev/kvm: Permission denied","pos":"virHostCPUGetCPUID:1422","subcomponent":"libvirt","thread":"38","timestamp":"2024-02-13T08:00:57.744000Z"}
{"component":"virt-launcher","level":"error","msg":"internal error: Unable to get session bus connection: Cannot spawn a message bus without a machine-id: Unable to load /var/lib/dbus/machine-id or /etc/machine-id: Failed to open file “/var/lib/dbus/machine-id”: No such file or directory","pos":"virGDBusGetSessionBus:126","subcomponent":"libvirt","thread":"38","timestamp":"2024-02-13T08:00:57.744000Z"}
{"component":"virt-launcher","level":"error","msg":"internal error: Unable to get system bus connection: Could not connect: No such file or directory","pos":"virGDBusGetSystemBus:99","subcomponent":"libvirt","thread":"38","timestamp":"2024-02-13T08:00:57.744000Z"}
{"component":"virt-launcher","level":"info","msg":"Connected to libvirt daemon","pos":"libvirt.go:579","timestamp":"2024-02-13T08:00:58.228416Z"}
{"component":"virt-launcher","level":"info","msg":"Registered libvirt event notify callback","pos":"client.go:562","timestamp":"2024-02-13T08:00:58.232549Z"}
{"component":"virt-launcher","level":"info","msg":"Marked as ready","pos":"virt-launcher.go:75","timestamp":"2024-02-13T08:00:58.232985Z"}
``` 

**kubectl get vmis -o yaml testvm**
```
apiVersion: kubevirt.io/v1
kind: VirtualMachine
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"kubevirt.io/v1","kind":"VirtualMachine","metadata":{"annotations":{},"name":"testvm","namespace":"default"},"spec":{"running":false,"template":{"metadata":{"labels":{"kubevirt.io/domain":"testvm","kubevirt.io/size":"small"}},"spec":{"domain":{"devices":{"disks":[{"disk":{"bus":"virtio"},"name":"containerdisk"},{"disk":{"bus":"virtio"},"name":"cloudinitdisk"}],"interfaces":[{"masquerade":{},"name":"default"}]},"resources":{"requests":{"memory":"64M"}}},"networks":[{"name":"default","pod":{}}],"volumes":[{"containerDisk":{"image":"quay.io/kubevirt/cirros-container-disk-demo"},"name":"containerdisk"},{"cloudInitNoCloud":{"userDataBase64":"SGkuXG4="},"name":"cloudinitdisk"}]}}}}
    kubevirt.io/latest-observed-api-version: v1
    kubevirt.io/storage-observed-api-version: v1
  creationTimestamp: "2024-02-13T08:00:34Z"
  finalizers:
  - kubevirt.io/virtualMachineControllerFinalize
  generation: 2
  name: testvm
  namespace: default
  resourceVersion: "155004"
  uid: 9fea2d80-5b08-489c-b4bd-e292e3ef7575
spec:
  running: true
  template:
    metadata:
      creationTimestamp: null
      labels:
        kubevirt.io/domain: testvm
        kubevirt.io/size: small
    spec:
      architecture: amd64
      domain:
        devices:
          disks:
          - disk:
              bus: virtio
            name: containerdisk
          - disk:
              bus: virtio
            name: cloudinitdisk
          interfaces:
          - masquerade: {}
            name: default
        machine:
          type: q35
        resources:
          requests:
            memory: 64M
      networks:
      - name: default
        pod: {}
      volumes:
      - containerDisk:
          image: quay.io/kubevirt/cirros-container-disk-demo
        name: containerdisk
      - cloudInitNoCloud:
          userDataBase64: SGkuXG4=
        name: cloudinitdisk
status:
  conditions:
  - lastProbeTime: "2024-02-13T08:00:54Z"
    lastTransitionTime: "2024-02-13T08:00:54Z"
    message: Guest VM is not reported as running
    reason: GuestNotRunning
    status: "False"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: null
    status: "True"
    type: LiveMigratable
  - lastProbeTime: null
    lastTransitionTime: "2024-02-13T08:00:59Z"
    message: 'failed to configure vmi network: setup failed, err: open /proc/sys/net/ipv4/ip_unprivileged_port_start:
      no such file or directory'
    reason: Synchronizing with the Domain failed.
    status: "False"
    type: Synchronized
  created: true
  desiredGeneration: 2
  observedGeneration: 2
  printableStatus: Starting
  volumeSnapshotStatuses:
  - enabled: false
    name: containerdisk
    reason: Snapshot is not supported for this volumeSource type [containerdisk]
  - enabled: false
    name: cloudinitdisk
    reason: Snapshot is not supported for this volumeSource type [cloudinitdisk]
``` 
**pod describe virt-launcher**
```
Name:             virt-launcher-testvm-q97zr
Namespace:        default
Priority:         0
Service Account:  default
Node:             node1/172.20.80.6
Start Time:       Tue, 13 Feb 2024 16:00:54 +0800
Labels:           kubevirt.io=virt-launcher
                  kubevirt.io/created-by=fd047462-f1b5-429c-98ac-f31dd30e1356
                  kubevirt.io/domain=testvm
                  kubevirt.io/nodeName=node1
                  kubevirt.io/size=small
                  vm.kubevirt.io/name=testvm
Annotations:      kubectl.kubernetes.io/default-container: compute
                  kubevirt.io/domain: testvm
                  kubevirt.io/migrationTransportUnix: true
                  kubevirt.io/vm-generation: 2
                  post.hook.backup.velero.io/command: ["/usr/bin/virt-freezer", "--unfreeze", "--name", "testvm", "--namespace", "default"]
                  post.hook.backup.velero.io/container: compute
                  pre.hook.backup.velero.io/command: ["/usr/bin/virt-freezer", "--freeze", "--name", "testvm", "--namespace", "default"]
                  pre.hook.backup.velero.io/container: compute
                  traffic.sidecar.istio.io/kubevirtInterfaces: k6t-eth0
Status:           Running
IP:               10.244.1.19
IPs:
  IP:           10.244.1.19
Controlled By:  VirtualMachineInstance/testvm
Init Containers:
  container-disk-binary:
    Container ID:  containerd://39cc9f0fa5665db2f6cb577e81f80db4f2c021cc74da4ba5898be2a147332555
    Image:         quay.io/kubevirt/virt-launcher:v1.1.1
    Image ID:      quay.io/kubevirt/virt-launcher@sha256:4fbfd76561667e98fafc75c0b804f77df251f96471d1121d69f149cf8fb9a4d8
    Port:          <none>
    Host Port:     <none>
    Command:
      /usr/bin/cp
      /usr/bin/container-disk
      /init/usr/bin/container-disk
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Tue, 13 Feb 2024 16:00:55 +0800
      Finished:     Tue, 13 Feb 2024 16:00:55 +0800
    Ready:          True
    Restart Count:  0
    Limits:
      cpu:     100m
      memory:  40M
    Requests:
      cpu:     10m
      memory:  1M
    Environment:
      XDG_CACHE_HOME:   /var/run/kubevirt-private
      XDG_CONFIG_HOME:  /var/run/kubevirt-private
      XDG_RUNTIME_DIR:  /var/run
    Mounts:
      /init/usr/bin from virt-bin-share-dir (rw)
  volumecontainerdisk-init:
    Container ID:  containerd://95f139e6afc6a2ec79ffe20e1fa47c4683becfff230e97c50405501cfe0d9501
    Image:         quay.io/kubevirt/cirros-container-disk-demo
    Image ID:      quay.io/kubevirt/cirros-container-disk-demo@sha256:0e5ac38b20abcc7752293425b239a147868facd62cd5030dede6da6f2fc526a1
    Port:          <none>
    Host Port:     <none>
 Command:
      /usr/bin/container-disk
    Args:
      --no-op
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Tue, 13 Feb 2024 16:00:56 +0800
      Finished:     Tue, 13 Feb 2024 16:00:56 +0800
    Ready:          True
    Restart Count:  0
    Limits:
      cpu:     10m
      memory:  40M
    Requests:
      cpu:                1m
      ephemeral-storage:  50M
      memory:             1M
    Environment:          <none>
    Mounts:
      /usr/bin from virt-bin-share-dir (rw)
      /var/run/kubevirt-ephemeral-disks/container-disk-data/fd047462-f1b5-429c-98ac-f31dd30e1356 from container-disks (rw)
Containers:
  compute:
    Container ID:  containerd://b8510f718729c9c530299dbfeda7c4f517973838b52a513814ccd105d6d6f029
    Image:         quay.io/kubevirt/virt-launcher:v1.1.1
    Image ID:      quay.io/kubevirt/virt-launcher@sha256:4fbfd76561667e98fafc75c0b804f77df251f96471d1121d69f149cf8fb9a4d8
    Port:          <none>
    Host Port:     <none>
    Command:
      /usr/bin/virt-launcher-monitor
      --qemu-timeout
      283s
      --name
      testvm
      --uid
      fd047462-f1b5-429c-98ac-f31dd30e1356
      --namespace
      default
      --kubevirt-share-dir
      /var/run/kubevirt
      --ephemeral-disk-dir
      /var/run/kubevirt-ephemeral-disks
      --container-disk-dir
      /var/run/kubevirt/container-disks
      --grace-period-seconds
      45
      --hook-sidecars
      0
      --ovmf-path
      /usr/share/OVMF
      --run-as-nonroot
    State:          Running
      Started:      Tue, 13 Feb 2024 16:00:57 +0800
    Ready:          True
    Restart Count:  0
    Limits:
      devices.kubevirt.io/kvm:        1
     devices.kubevirt.io/tun:        1
      devices.kubevirt.io/vhost-net:  1
    Requests:
      cpu:                            100m
      devices.kubevirt.io/kvm:        1
      devices.kubevirt.io/tun:        1
      devices.kubevirt.io/vhost-net:  1
      ephemeral-storage:              50M
      memory:                         317880392
    Environment:
      XDG_CACHE_HOME:   /var/run/kubevirt-private
      XDG_CONFIG_HOME:  /var/run/kubevirt-private
      XDG_RUNTIME_DIR:  /var/run
      POD_NAME:         virt-launcher-testvm-q97zr (v1:metadata.name)
    Mounts:
      /var/run/kubevirt from public (rw)
      /var/run/kubevirt-ephemeral-disks from ephemeral-disks (rw)
      /var/run/kubevirt-private from private (rw)
      /var/run/kubevirt/container-disks from container-disks (rw)
      /var/run/kubevirt/hotplug-disks from hotplug-disks (rw)
      /var/run/kubevirt/sockets from sockets (rw)
      /var/run/libvirt from libvirt-runtime (rw)
  volumecontainerdisk:
    Container ID:  containerd://0e012cd9ac98b26d835f9d9fb93f66d7a27d353def706b535019e55af41db32e
    Image:         quay.io/kubevirt/cirros-container-disk-demo
    Image ID:      quay.io/kubevirt/cirros-container-disk-demo@sha256:0e5ac38b20abcc7752293425b239a147868facd62cd5030dede6da6f2fc526a1
    Port:          <none>
    Host Port:     <none>
    Command:
      /usr/bin/container-disk
    Args:
      --copy-path
      /var/run/kubevirt-ephemeral-disks/container-disk-data/fd047462-f1b5-429c-98ac-f31dd30e1356/disk_0
    State:          Running
      Started:      Tue, 13 Feb 2024 16:00:58 +0800
    Ready:          True
    Restart Count:  0
    Limits:
      cpu:     10m
      memory:  40M
    Requests:
      cpu:                1m
      ephemeral-storage:  50M
      memory:             1M
    Environment:          <none>
    Mounts:
      /usr/bin from virt-bin-share-dir (rw)
      /var/run/kubevirt-ephemeral-disks/container-disk-data/fd047462-f1b5-429c-98ac-f31dd30e1356 from container-disks (rw)
  guest-console-log:
    Container ID:  containerd://95f65446848085548335404310c0a0fa0de2aeb430a557eff0d2ec26dcb50eb7
    Image:         quay.io/kubevirt/virt-launcher:v1.1.1
    Image ID:      quay.io/kubevirt/virt-launcher@sha256:4fbfd76561667e98fafc75c0b804f77df251f96471d1121d69f149cf8fb9a4d8
    Port:          <none>
    Host Port:     <none>
    Command:
      /usr/bin/virt-tail
    Args:
      --logfile
      /var/run/kubevirt-private/fd047462-f1b5-429c-98ac-f31dd30e1356/virt-serial0-log
State:          Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Tue, 13 Feb 2024 16:00:59 +0800
      Finished:     Tue, 13 Feb 2024 16:01:19 +0800
    Ready:          False
    Restart Count:  0
    Limits:
      cpu:     15m
      memory:  60M
    Requests:
      cpu:     5m
      memory:  35M
    Environment:
      VIRT_LAUNCHER_LOG_VERBOSITY:  2
    Mounts:
      /var/run/kubevirt-private from private (ro)
Readiness Gates:
  Type                                   Status
  kubevirt.io/virtual-machine-unpaused   True
Conditions:
  Type                                   Status
  kubevirt.io/virtual-machine-unpaused   True
  Initialized                            True
  Ready                                  False
  ContainersReady                        False
  PodScheduled                           True
Volumes:
  private:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  public:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  sockets:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  virt-bin-share-dir:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  libvirt-runtime:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  ephemeral-disks:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  container-disks:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  hotplug-disks:
    Type:        EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:   <unset>
QoS Class:       Burstable
Node-Selectors:  kubernetes.io/arch=amd64
                 kubevirt.io/schedulable=true
Tolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                 node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  47s   default-scheduler  Successfully assigned default/virt-launcher-testvm-q97zr to node1
  Normal  Pulled     46s   kubelet            Container image "quay.io/kubevirt/virt-launcher:v1.1.1" already present on machine
  Normal  Created    46s   kubelet            Created container container-disk-binary
  Normal  Started    46s   kubelet            Started container container-disk-binary
  Normal  Pulling    46s   kubelet            Pulling image "quay.io/kubevirt/cirros-container-disk-demo"
  Normal  Pulled     45s   kubelet            Successfully pulled image "quay.io/kubevirt/cirros-container-disk-demo" in 942ms (942ms including waiting)
  Normal  Created    45s   kubelet            Created container volumecontainerdisk-init
  Normal  Started    45s   kubelet            Started container volumecontainerdisk-init
  Normal  Pulled     44s   kubelet            Container image "quay.io/kubevirt/virt-launcher:v1.1.1" already present on machine
  Normal  Created    44s   kubelet            Created container compute
  Normal  Started    44s   kubelet            Started container compute
  Normal  Pulling    44s   kubelet            Pulling image "quay.io/kubevirt/cirros-container-disk-demo"
  Normal  Pulled     43s   kubelet            Successfully pulled image "quay.io/kubevirt/cirros-container-disk-demo" in 996ms (996ms including waiting)
  Normal  Created    43s   kubelet            Created container volumecontainerdisk
  Normal  Started    43s   kubelet            Started container volumecontainerdisk
  Normal  Pulled     43s   kubelet            Container image "quay.io/kubevirt/virt-launcher:v1.1.1" already present on machine
  Normal  Created    43s   kubelet            Created container guest-console-log
  Normal  Started    43s   kubelet            Started container guest-console-log
```

**Comment by xpivarc** [comment]
Hey @hjxgitcode ,
Please provide logs of virt-handler.

**Comment by hjxgitcode** [comment]
> Hey @hjxgitcode , Please provide logs of virt-handler.
Isn't that the top log? What command should I use to view the log？

**Comment by xpivarc** [comment]
The first log is from virt-launcher. A virt-handler is a daemonset.

**Comment by hjxgitcode** [comment]
> The first log is from virt-launcher. A virt-handler is a daemonset.

This is too much

**Comment by hjxgitcode** [comment]
I have solved this problem.The problem is that the Linux kernel version is low, and I resolved it by updating from 3.1xx to 5.4xx.

**Comment by Nilesh1206** [comment]
I faced the same issue on the Vanilla cluster, but it worked on the OpenShift Cluster.
